src/python/AsyncStageOut/__init__.py

```diff
@@ -57,3 +59,57 @@ def getDNFromUserName(username, log):
        return dn
     return dn
 
+se_site_map = {}
+se_site_map_last_update = None
+
+def getSiteNamesFromSENames(sites, log, force=False):
+    """
+    When injecting from the WN, the CMS site name for the temporary stageout location is not known.
+    Only the SE name is know.
```

Comment:
This is not true. You should have the PNN (PhEDEx node name) available as well in the TFC or site local config. That was the whole point of this transition. 

There is WMCore code to do this. If you don't find it, let me know.